# Day 6 - Advanced Analytics

1. Historical VaR & CVaR
2. Rolling Sharpe Ratio
3. Investor Cohort Analysis
4. SIP Continuity Analysis
5. Fund Recommendation
6. Sector HHI Concentration

In [2]:
import pandas as pd

var_df = pd.read_csv("../data/processed/var_cvar_report.csv")
cohort_df = pd.read_csv("../data/processed/cohort_analysis.csv")
sip_df = pd.read_csv("../data/processed/sip_continuity.csv")
hhi_df = pd.read_csv("../data/processed/sector_hhi.csv")

var_df.head()

,amfi_code,var_95_pct,cvar_95_pct,var_cvar_ratio,ann_vol_pct,skewness,excess_kurtosis,tail_days_count,total_days,scheme_name,fund_house,sub_category,risk_category,plan,var_rank
0,120507,-0.0222,-0.0373,1.6815,0.4937,-0.0181,-0.0863,58,1149,ICICI Pru Liquid Fund - Regular - Growth,ICICI Prudential MF,Liquid,Low,Regular,1
1,101208,-0.0269,-0.0422,1.5711,0.5062,-0.0385,-0.0579,58,1149,ABSL Liquid Fund - Regular - Growth,Aditya Birla Sun Life MF,Liquid,Low,Regular,2
2,120844,-0.0285,-0.0411,1.4431,0.5131,0.0086,-0.0580,58,1149,Kotak Liquid Fund - Regular - Growth,Kotak Mahindra MF,Liquid,Low,Regular,3
3,100025,-0.3793,-0.4994,1.3165,3.9035,-0.0195,0.2531,58,1149,HDFC Short Term Debt Fund - Regular - Growth,HDFC Mutual Fund,Short Duration,Low,Regular,4
4,118636,-0.3804,-0.4919,1.2930,3.9838,-0.0010,-0.0773,58,1149,Nippon India Gilt Securities Fund - Regular - ...,Nippon India MF,Gilt,Low,Regular,5


## Key Insights

1. SBI Small Cap Fund - Direct Plan showed the highest downside risk with VaR of -2.686%.

2. ICICI Prudential Liquid Fund was the safest scheme with VaR of only -0.022%.

3. The 2024 investor cohort contributed over ₹349 Cr of investments.

4. 97.8% of eligible SIP investors were flagged as at-risk due to large gaps between investments.

5. Axis Bluechip Fund had the highest sector concentration (HHI = 2967.7), while UTI Mid Cap Fund was the most diversified.